app/src/components/PlayPauseButton.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react'
+import MainButton from '@components/MainButton'
+
+interface PlayPauseButtonProps {
+  paused: boolean
```

Comment:
I would rename this to `initialState` or something similar so it does not get confused with the local state `isPaused`.